src/components/mols/CreditCardPayment/index.tsx

```diff
@@ -0,0 +1,250 @@
+"use client";
+import React, { useState } from "react";
+import { Formik, Form, Field } from "formik";
+import { Input, ButtonPrimary, FormItem, Select } from "@/components";
+import { ICreditCardPayment } from "@/types/payment";
+import * as Yup from "yup";
+import valid from "card-validator";
+import Image from "next/image";
+import CardFlag from "@/images/easytolive/payment/card-flag.svg";
+
+interface ICreditCardPaymentProps {
+  currentStepPayment: React.Dispatch<React.SetStateAction<number>>;
+}
+
+const CreditCardPayment: React.FC<ICreditCardPaymentProps> = ({
+  currentStepPayment,
+}) => {
+  const [loading, setLoading] = useState(false);
+  const [formattedCardNumber, setFormattedCardNumber] = useState("");
+
+  const Months = [
+    " - JAN",
+    " - FEV",
+    " - MAR",
+    " - ABR",
+    " - MAI",
+    " - JUN",
+    " - JUL",
+    " - AGO",
+    " - SET",
+    " - OUT",
+    " - NOV",
+    " - DEZ",
+  ];
```

Comment:
Primeiro move essa constante pra uma outra pasta tipo /utils/consts, nao sei qual exatamente

```suggestion
  const MONTHS = [
    "JAN",
    "FEV",
    "MAR",
    ...
  ];
```
```
MONTHS.map((eachMonth, idx) => 
  <option value={idx}>{idx.toString().concat(' - ').concat(eachMonth)}</option>
)
```

Reply:
Aulas ok?

Reply:
corrigido